xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -390,8 +390,6 @@ namespace XBMCAddon
             item->GetVideoInfoTag()->m_strProductionCode = value;
           else if (key == "aired")
             item->GetVideoInfoTag()->m_firstAired.SetFromDateString(value);
-          else if (key == "credits")
-            item->GetVideoInfoTag()->m_writingCredits = StringUtils::Split(value, g_advancedSettings.m_videoItemSeparator);
```

Comment:
Probably not, no need to make fun of my C++ skills ;)
But I think I know all sides (core / script api / skin api / json api) well enough to know how to get our  existing stuff into a consistent shape.